site/src/components/image.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Image
+description: Image component in the Atlas Design System
+template: standard
+---
+
+# Image
+
+The image component parents an image tag or svg and provides width based classes to contrain its children.
+
+## Usage
+
+Here is an example of a standard usage of label paired with an input. The default image class takes up the full width of its container.
+
+```html
+<div class="image">
+	<img src="~/src/scaffold/media/video-media.png" />
+</div>
+```
+
+## Sizes
+
+The following classes are available for resizing square images.
```

Comment:
What do you think about including the small table and listing all the available sizes? I think it might be easier than checking the code example

Reply:
Added, see what you think

Reply:
Do you think people might want to mix sizes and do smth like `16x32` by seeing this pattern? Would it be better to list sizes as `16x16, 24x24, etc`? Not insisting, just sharing thoughts.

Reply:
I don't think we should open pixel sizing up to non-square ratios, and the description mentions that they're square

Reply:
You know what, nevermind. I like how explicit your idea is, going to update